documentation/docs/08-events.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Events
+---
+
+SvelteKit emits [CustomEvents](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent) on the `window` object when certain things happen:
+
+* `sveltekit:start` — fired once at the start of the app's life
```

Comment:
I think it would be clearer here to say once the app has hydrated, or whatever else happens. 'at the start of the app's life' could mean lots of things.

Reply:
yeah, true. fixed